src/pages/Kvittering.tsx

```diff
@@ -9,11 +10,20 @@ const Kvittering = () => {
   return (
     <div className="limit">
       <AlertStripeInfo>
-        <div>Søknaden er mottat</div>
-        <div>Referansnummer: {referanseNummer}</div>
+        <div>
+          <Ingress>Søknaden er mottatt.</Ingress>
+          Dersom den blir godkjent vil beløpet utbetales innen X virkedager. Beløpet kan ikke overstige 6G.
+        </div>
+        <div>
+          <Ingress>Referansenummer: <span style={{fontWeight: "bold"}}>{referanseNummer}</span></Ingress>
```

Comment:
Du kan bruke <B> for bold. Vet ikke om dette betyr noe annerledes for skjermledere men det er mer riktig.

Reply:
og så så mye penere!

Reply:
Generelt skal man helt unngå inline styling hvis det er mulig :)